app/grandchallenge/challenges/models.py

```diff
@@ -1,5 +1,6 @@
 import datetime
 import logging
+from functools import cached_property
```

Comment:
I think that you want to use the one from django, not functools.

Reply:
Oh wait, why was that import changed?

Reply:
Not sure actually! Using the one from django now.